### PR TITLE
[Tabs] readme update.

### PR DESCRIPTION
**Pull request by mohammadcazig**

### CODE REVIEW

File: components/Tabs/README.md [hunk 1]
```diff
@@ -2,72 +2,98 @@
...
-<div class="article__asset article__asset--screenshot">
-  <img src="docs/assets/tabs.png" alt="Tabs" width="375">
-</div>
+# Tabs
 
 Tabs are bars of buttons used to navigate between groups of content.
 
-## Design & API Documentation
+<img src="docs/assets/tabs.png" alt="Tabs" width="375">
+
```
Comment (jverkoey) [inline components/Tabs/README.md]: Need to keep the screenshot surrounding tags.

File: components/Tabs/docs/Examples.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+### Creating a tab bar
```
Comment (jverkoey) [inline components/Tabs/docs/Examples.md:1]: Lower case hyphenated names here and throughout please. This is following the web url convention.

**Review by jverkoey — CHANGES_REQUESTED** [review]

File: components/Tabs/docs/Examples.md (hunk 2 above)
Comment (mohammadcazig) [reply components/Tabs/docs/Examples.md:1]: done.

File: components/Tabs/README.md (hunk 1 above)
Comment (mohammadcazig) [reply components/Tabs/README.md]: done.

Comment (mohammadcazig) [reply components/Tabs/README.md]: done.

File: components/Tabs/docs/Examples.md (hunk 2 above)
Comment (mohammadcazig) [reply components/Tabs/docs/Examples.md:1]: done.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**